### ISSUE TITLE
[Image needs adding] Research Ethics Committees Chapter

### ISSUE THREAD

**Issue by Arielle-Bennett** [the-turing-way/the-turing-way#3345]
### Summary

The chapter: https://the-turing-way.netlify.app/ethical-research/ethics-committees does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

### What needs to be done?

- [x] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [x] Write the alt text for the image
- [x] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*
